docs/guides/nrfconnect_android_commissioning.md

```diff
@@ -0,0 +1,176 @@
+[nRF Connect Lock Example Application](../../examples/lock-app/nrfconnect/README.md) onto a CHIP-enabled Thread network.
+
+Prerequisites:
+- 2 x nRF 52840 DK
+- 1 x smartphone with Android 8+
+- 1 x PC with Ubuntu 20.04 and Docker installed (however, all the presented steps should be achievable in some way using other popular operating systems)
+
+The essence of the commissioning process is composed of the following steps:
+1. CHIPTool discovers the accessory device over BLE.
+2. CHIPTool establishes a secure connection to the device over BLE and sends Thread credentials.
+3. The accessory device joins a Thread network.
+
+Since a typical smartphone doesn't have native support for Thread networking, extra effort is needed to prepare the test environment. The article describes how to turn a PC with spare Wi-Fi card and nRF 52840 DK into a Thread border router to forward Wi-Fi packets to a Thread network. The below diagram depicts connectivity between CHIPTool and Lock Example Application:
+
+```
+               +--------------------+
+               |    Smartphone      |
+     +---------|  Android CHIPTool  |---------+
+     |         +--------------------+         |
+     |                                        | Wi-Fi
+     |                                        |
+     |                                +---------------+  Ethernet  +----------+
+     |                                |      PC       |------------| Internet |
+     |                                +---------------+            +----------+
+     |                                        |
+ BLE |                                        | USB
+     |                                        |
+     |                                +---------------+
+     |                                | nRF 52840 DK  |
+     |                                | OpenThread RCP|
+     |                                +---------------+
+     |                                        |
+     |         +--------------------+         | Thread
+     |         |    nRF 52840 DK    |         |
+     +---------|  Lock Application  |---------+
+               +--------------------+
+```
+## Building and flashing OpenThread RCP firmware
+
+<details>
+<summary>Click here to expand</summary>
+<br>
+OpenThread RCP firmware is required to allow a PC to communicate with a Thread network. Complete the following steps to build and flash the RCP firmware onto a nRF 52840 DK:
+
+1. Download and install the
+[nRF Command Line Tools](https://www.nordicsemi.com/Software-and-Tools/Development-Tools/nRF-Command-Line-Tools).
+2. Clone OpenThread repository into the current directory:
+
+        $ git clone https://github.com/openthread/openthread.git
+        
+3. Install OpenThread dependencies:
+
+        $ cd openthread
+        $ ./script/bootstrap
+4. Set up build environment:
+
+        $ ./bootstrap
+5. Build OpenThread for the nRF 52840 DK:
+
+        $ make -f examples/Makefile-nrf52840
```

Comment:
Are you sure that building without COMMISSIONER=1 flag will work?

Reply:
Yes

Reply:
It should, RCP is just radio transceiver.